frontend/packages/dev-console/src/components/topology/components/nodes/KnativeServiceGroup.tsx

```diff
@@ -92,7 +92,12 @@ const KnativeServiceGroup: React.FC<KnativeServiceGroupProps> = ({
   }, [editAccess, innerHover, onShowCreateConnector, onHideCreateConnector]);
 
   return (
-    <Tooltip content="Move sink to service" trigger="manual" isVisible={dropTarget && canDrop}>
+    <Tooltip
+      content="Move sink to service"
+      trigger="manual"
+      isVisible={dropTarget && canDrop}
+      tippyProps={{ duration: 0, delay: 0 }}
```

Comment:
Can you confirm the behavior with UX? 0 delay will bring up tooltips immediately as you hover your mouse over an intermediate node.

Didn't we try avoid this kind of behavior for label truncation by having a small delay.

Reply:
UX spec specifies `For connectors, the tooltip should appear the moment the user hovers over a valid drop target.`